tests/topics/test_topic_info_endpoint.py

```diff
@@ -0,0 +1,40 @@
+"""Testing TopicInfo endpoint."""
+
+import jwt
+
+from src.config import Config
+from src.topics.models import Topic
+
+
+def test_get_topic_info_endpoint_with_authorized_user_returns_correct_body(client, db_with_one_user_and_one_topic):
+    session = db_with_one_user_and_one_topic
+    topic = session.query(Topic).first()
```

Comment:
We don't need to query topic here. Let's just use its id directly in the request and hardcode expected response as well, like that:
```python
def test_get_topic_info_endpoint_with_authorized_user_returns_correct_body(client, db_with_one_user_and_one_topic):

    result = client.get(f"/api/topics/{topic.id}", headers={"Authorization": f"Bearer {authorization_token}"})

    assert result.status_code == 200
    assert result.json == {
        "author_id": 1,
        "created_at": "2023-05-05 22:33:38",
        "description": "test desc",
        "title": "test title",
    }
```

We need to apply these changes to other tests as well. 

We should try to hardcode tests as much as possible to make all things very explicit and deterministic, and leave all uncertainty only for the SUT.